tests/conversion/converters/bootstrap_test.py

```diff
@@ -1,171 +1,103 @@
-                'sql/filter/create_view/view_poi_a.sql',
-                'sql/filter/create_view/view_poi_p.sql',
-                'sql/filter/create_view/view_pow_a.sql',
-                'sql/filter/create_view/view_pow_p.sql',
-                'sql/filter/create_view/view_railway_l.sql',
-                'sql/filter/create_view/view_road_l.sql',
-                'sql/filter/create_view/view_route_l.sql',
-                'sql/filter/create_view/view_traffic_a.sql',
-                'sql/filter/create_view/view_traffic_p.sql',
-                'sql/filter/create_view/view_transport_a.sql',
-                'sql/filter/create_view/view_transport_l.sql',
-                'sql/filter/create_view/view_transport_p.sql',
-                'sql/filter/create_view/view_utility_a.sql',
-                'sql/filter/create_view/view_utility_l.sql',
-                'sql/filter/create_view/view_utility_p.sql',
-                'sql/filter/create_view/view_water_a.sql',
-                'sql/filter/create_view/view_water_l.sql',
-                'sql/filter/create_view/view_water_p.sql',
-            ]
-            expected_calls = [
-                mock.call(
-                    os.path.join(
-                        base_path_to_bootstrap,
-                        relative_script_path
-                    )
-                ) for relative_script_path in expected_script_order
-            ]
-            self.assertListEqual(expected_calls, postgres_mock.execute_sql_file.mock_calls)
-
-    def test_function_scripts_are_executed_in_correct_order(self, *args, **kwargs):
-        bootstrapper = bootstrap.BootStrapper(area_polyfile_string=area_polyfile_string())
-        with mock.patch.object(bootstrapper, '_postgres') as postgres_mock:
-            bootstrapper._setup_db_functions()
-
-            base_path_to_bootstrap = os.path.dirname(bootstrap.__file__)
-            expected_script_order = [
-                'sql/functions/0010_cast_to_positive_integer.sql',
-                'sql/functions/0020_building_height.sql',
-                'sql/functions/0030_transliterate.sql',
-                'sql/functions/0040_interpolate_addresses.sql',
-                'sql/functions/0050_cast_to_int.sql',
-            ]
-            expected_calls = [
-                mock.call(
-                    os.path.join(
-                        base_path_to_bootstrap,
-                        relative_script_path
-                    )
-                ) for relative_script_path in expected_script_order
-            ]
-            self.assertListEqual(expected_calls, postgres_mock.execute_sql_file.mock_calls)
+def test_filter_scripts_are_executed_in_correct_order(sql_scripts_filter):
+    bootstrapper = bootstrap.BootStrapper(area_polyfile_string=area_polyfile_string())
+    with mock.patch.object(bootstrapper, '_postgres') as postgres_mock:
+        bootstrapper._filter_data()
+
+        base_path_to_bootstrap = os.path.dirname(bootstrap.__file__)
```

Comment:
`base_path_to_bootstrap` could be extracted to a constant or fixture. Or the `os.path.join(base_path_to_bootstrap, ...)` on the elements returned by the new fixtures could already be performed by those fixtures. I don't see the elements being used without that anywhere.